wave_kinematics/include/wave/kinematics/two_wheel.hpp

```diff
@@ -19,7 +19,7 @@ class TwoWheelRobot2DModel {
     Vec3 pose;
 
     TwoWheelRobot2DModel() : pose{0.0, 0.0, 0.0} {}
-    TwoWheelRobot2DModel(const Vec3 &pose) : pose{pose} {}
+    explicit TwoWheelRobot2DModel(const Vec3 &pose) : pose{pose} {}
```

Comment:
Yep. It is actually part of [google style guide](https://google.github.io/styleguide/cppguide.html#Implicit_Conversions)

clang-tidy highlighted it and also said "pass by value and use std::move" for the argument, which  makes sense but I just left it for now.